docs/proposals/backup-restore.md

```diff
@@ -121,18 +121,35 @@ oras backup --output backup.tar --include-referrers registry-a.k8s.io/kube-apise
 Alternatively, `oras` should also enables users to choose which tags within a repository to back up. For example, back up `v1` and `v2` tags with their referrers:
 
 ```bash
-oras backup registry-a.k8s.io/kube-apiserver:v1,v2 --include-referrers --output airgap-snapshot.tar
+oras backup registry-a.k8s.io/kube-apiserver:v1,v2 --include-referrers --output backup.tar
 ```
 
 Upon success, the output will be:
 
 ```console
-Found 2 tag(s) in registry-a.k8s.io/kube-apiserver
-- v1
-- v2
-<progress bar>...
-Exporting to backup.tar
-Successfully backed up 2 tag(s) from registry-a.k8s.io/kube-apiserver to backup.tar
+## <progress_bar>
+✔ Pulled tag v1 and 1 referrer(s)
```

Comment:
Is the referrer information important as its implementation cost a lot.

Reply:
Just an update: its implementation won't cost a lot as we can get the info from the local copy. So... it's up to you.